shelley/design-spec/delegation_design_spec.tex

```diff
@@ -799,23 +800,38 @@ \subsubsection{Bootstrap Address}
 \subsubsection{HD Wallet Structure in Shelley}
 \label{hd-wallet-structure-in-shelley}
 
-All the Shelley address formats (other than script addresses) support
-hierarchical deterministic wallets, as per BIP-32 \citep{bip32}.
+The Shelley addresses where the value address is a single public key hash
+support hierarchical deterministic wallets, as per BIP-32~\citep{bip32}.
+
+For multi-sig value addresses, we can use a slight generalisation of
```

Comment:
For all my going on about script addresses, yes it's absolutely true that for a wallet it has to manage its addresses knowing the particular thing its doing with those scripts: i.e. a multi-sig account. So it's fine for the parts of the doc that talk about chain validity to be general and talk about scripts, and for the wallet part to be very specific to particular patterns/templates of multi-sig scripts.